src/api/cpp/cvc5_types.h

```diff
@@ -28,15 +28,28 @@ namespace cvc5 {
  */
 enum UnknownExplanation
 {
+  /**
+   * Full satisfiability check required (e.g., if only preprocessing was
+   * performed).
+   */
   REQUIRES_FULL_CHECK,
+  /** Incomplete theory. */
```

Comment:
```suggestion
  /** Incomplete theory solver. */
```

is maybe a bit more accurate?